src/Action/LoginAction.php

```diff
@@ -86,12 +93,13 @@ public function __construct(
         $this->templateRegistry = $templateRegistry;
         $this->tokenStorage = $tokenStorage;
         $this->session = $session;
+        $this->translator = $translator;
     }
 
     public function __invoke(Request $request): Response
     {
         if ($this->isAuthenticated()) {
-            $this->session->getFlashBag()->add('sonata_user_error', 'sonata_user_already_authenticated');
+            $this->session->getFlashBag()->add('sonata_user_error', $this->translator->trans('sonata_user_already_authenticated', [], 'SonataUserBundle'));
```

Comment:
And here make sure `$this->translator` is not null

Reply:
I went with showing the untranslated message, not sure if it would be better to show no message at all...